common/reviews/api/rush-lib.api.md

```diff
@@ -602,6 +604,8 @@ export class PackageJsonEditor {
     // (undocumented)
     tryGetDevDependency(packageName: string): PackageJsonDependency | undefined;
     // (undocumented)
+    tryGetResolution(packageName: string): PackageJsonDependency | undefined;
```

Comment:
Is this name clear?

Reply:
i test remove in yarn, and yarn will not remove project in `resolutions`, so i delete this function, now rush remove will not remove anything in resolution.and i think this logic is more similar to 'rush update' or 'rush add'